### PR TITLE
Add more tests to tizen_device_test.dart

### PR DESCRIPTION
**Pull request by swift-kim**
Add basic tests for `TizenDevice.installApp()`, `TizenDevicePortForwarder`.

### CODE REVIEW

File: test/general/tizen_device_test.dart [hunk 1]
```diff
@@ -207,6 +280,61 @@ void main() {
...
+  testWithoutContext(
+      'TizenDevicePortForwarder.forwardedPorts can list forwarded ports', () {
+    final TizenDevicePortForwarder forwarder = TizenDevicePortForwarder(
+      device: _createTizenDevice(
+        processManager: processManager,
+        fileSystem: fileSystem,
+      ),
+      logger: logger,
+    );
+    processManager.addCommands(<FakeCommand>[
```
Comment (bbrto21) [inline test/general/tizen_device_test.dart:293]: If you use `addCommand` instead of `addCommand`s, this part can be shorter.

**Review by bbrto21 — APPROVED** [review]
Generally LGTM.

File: test/general/tizen_device_test.dart (hunk 1 above)
Comment (swift-kim) [reply test/general/tizen_device_test.dart:293]: Oh, you're right. I just forgot.

File: test/general/tizen_device_test.dart [hunk 2]
```diff
@@ -106,6 +110,75 @@ void main() {
...
+        command: _sdbCommand(<String>['uninstall', 'TestPackage']),
+        stdout: '''
+__return_cb req_id[1] pkg_type[tpk] pkgid[TestPackage] key[install_percent] val[100]
+__return_cb req_id[1] pkg_type[tpk] pkgid[TestPackage] key[end] val[ok]
+''',
+      ),
+      FakeCommand(command: _sdbCommand(<String>['install', 'app.tpk'])),
+    ]);
+
+    expect(await device.installApp(tpk), isTrue);
```
Comment (JSUYA) [inline test/general/tizen_device_test.dart:176]: How about setting the value of `apiVersion` in `_FakeTizenManifest`?
You can increase the coverage of the `installApp()` function by one more line.

[tizen_device.dart]
```
     260                 :          2 :     final Version? platformVersion = Version.parse(_platformVersion);
     261                 :          3 :     final Version? apiVersion = Version.parse(app.manifest.apiVersion);
     262                 :            :     if (platformVersion != null &&
     263                 :            :         apiVersion != null &&
     264                 :          0 :         apiVersion > platformVersion) {   // <---- here
     265                 :          0 :       _logger.printWarning(
```

**Review by JSUYA — APPROVED** [review]

File: test/general/tizen_device_test.dart (hunk 2 above)
Comment (swift-kim) [reply test/general/tizen_device_test.dart:212]: Thanks. I added a test for the version comparison.

**Pull request merged**